### PR TITLE
docs(fluid-static): Add explicit release tags to exported API members

### PR DESCRIPTION
**Pull request by Josmithr**
Removes temporary workaround in package api-extractor config

[AB#5896](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/5896)

### CODE REVIEW

File: packages/framework/fluid-static/src/types.ts [hunk 1]
```diff
@@ -219,6 +245,8 @@ export interface IMember {
 }
 
 /**
- * An extended member object that includes currentConnection
+ * An extended member object that includes currentConnection.
+ *
+ * @public
  */
 export type Myself<M extends IMember = IMember> = M & { currentConnection: string };
```
Comment (Josmithr) [inline packages/framework/fluid-static/src/types.ts:252]: @zhenmichael While I'm here, I'm wondering if we can add a bit more context to this type. Looks like you maybe added this one? Can you offer some extra context on what `currentConnection` is?

Comment (zhenmichael) [reply packages/framework/fluid-static/src/types.ts:252]: @Josmithr currentConnection is the container clientId associated with the member instance. Specifically, it is returned as this.container.clientId where container is defined in a ServiceAudience constructor.

**Review by DLehenbauer — CHANGES_REQUESTED** [review]
Container seems like a safe bet, but let's wait for guidance before committing @public to the repo.

@microsoft/fluid-cr-api - FYI

File: packages/framework/fluid-static/src/types.ts (hunk 1 above)
Comment (Josmithr) [reply packages/framework/fluid-static/src/types.ts:252]: We don't appear to expose the `clientId` concept in the `IFluidContainer` API. Is there a more explicit way of expressing the semantics of `currentConnection` apart from the `IContainer` implementation details?

**Comment by Josmithr** [comment]
Closing this since we have opted to take a different path towards ensuring all our APIs are tagged.

**Pull request closed without merging**